src/literals.py

```diff
@@ -35,3 +39,25 @@ class Ports:
     "SASL_SSL": Ports(9093, 19093),
     "SSL": Ports(9094, 19094),
 }
+
+
+class Status(Enum):
+    ACTIVE = ActiveStatus()
+    NO_PEER_RELATION = WaitingStatus("no peer relation yet")
+    SNAP_NOT_INSTALLED = BlockedStatus(f"unable to install {SNAP_NAME} snap")
+    SNAP_NOT_RUNNING = BlockedStatus("snap service not running")
```

Comment:
it'd be quite good to give the user a hint at what to do (enable and start snapd)